toolkit/pallets/block-production-log/src/mock.rs

```diff
@@ -62,13 +62,26 @@ impl frame_system::Config for Test {
 	type PostTransactions = ();
 }
 
+#[cfg(feature = "runtime-benchmarks")]
+pub struct PalletBlockProductionLogBenchmarkHelper;
+
+#[cfg(feature = "runtime-benchmarks")]
+impl crate::benchmarking::BenchmarkHelper<[u8; 32]> for PalletBlockProductionLogBenchmarkHelper {
+	fn producer_id() -> [u8; 32] {
+		Default::default()
+	}
+}
+
 impl crate::pallet::Config for Test {
 	type BlockProducerId = [u8; 32];
 	type WeightInfo = ();
 
 	fn current_slot() -> Slot {
 		Slot::from(1001000)
 	}
+
+	#[cfg(feature = "runtime-benchmarks")]
+	type BenchmarkHelper = PalletBlockProductionLogBenchmarkHelper;
```

Comment:
if it will have impact only on benchmarks, maybe there is no need to make a parametrization and just hardcode some valid address in benchmarks code

Reply:
I would like to keep it like in polkadot-sdk, where every definition is parameterized. Maybe without parametrization it would signal some unused code?

Reply:
yep thats possible